gym/envs/toy_text/kellycoinflip.py

```diff
@@ -93,7 +93,7 @@ def __init__(self, initialWealth=25, edgePriorAlpha=7, edgePriorBeta=3, maxWealt
         self.maxRounds = maxRounds
         self.rounds = self.maxRounds
         self.maxWealth = maxWealth
-        self._seed()
+        if reseed: self._seed()
```

Comment:
So to do a check, you could change the `__init__` line to 

```.python
if reseed or not hasattr(self, 'np_random') : self._seed()
```

I think that would work, if I've understood what `hasattr` does right.